<launch>
	<!-- Artag pose -->
	<node name="artag_pose" type="artag_pose_node" pkg="artag_pose" output="screen">
	    <param name="hz" type="double" value="10.0" />
	    <param name="useYAML" type="boolean" value="false" />
		<param name="configFileName" type="string" value="$(find artag_pose)/config/cafeteria.yaml" />
		<rosparam command="load" file="$(find artag_pose)/config/cafeteria.yaml" />
	</node>
</launch>
